lib/resources/r4/appointment.yaml

```diff
@@ -101,3 +101,41 @@ fields:
     }
```

Comment:
@MaxPhilips Are we missing any terminologies for Participant?

I would at least expect this one: https://www.hl7.org/fhir/R4/valueset-encounter-participant-type.html

Reply:
Discussed with Kathy, added participant.type terminology mapping to 14250
https://github.com/cerner/fhir.cerner.com/pull/269/commits/c0e31f83d97a586fe86cd658b1ec8f95ccb6af75

Reply:
I talked with Kathy and Max this morning, we need to document the HL7 ValueSet for participant.type (which I posted above).

We also need to document:

- reasonCode
- participant.reason
- participant.status

Reply:
as discussed, added no binding doc for reasonCode,
participant.required and participant.status have required binding, so no doc added.
ea6a1d5

Reply:
+1 after resolved comment